testssl.sh

```diff
@@ -12436,6 +12471,12 @@ run_grease() {
 
      # Send a list of non-existent ciphers where the second byte does not match
      # any existing cipher.
+
+     # Need to ensure that $TEMPDIR/$NODEIP.parse_tls_serverhello.txt contains the results of the
+     # most recent calls to tls_sockets even if tls_sockets is not successful. Setting $DEBUG to
+     # a non-zero value ensures this. Setting it to 1 prevents any extra information from being
+     # displayed.
+     [[ $DEBUG -eq 0 ]] && DEBUG=1
```

Comment:
That seems not correct to me. DEBUG is a global

Reply:
This is just a temporary change to the value of $DEBUG. It gets set back to its original value about 20 lines later.

As noted above and in the comments in the code, there is one set of tests in run_grease() that need to be able to read `$TEMPDIR/$NODEIP.parse_tls_serverhello.txt` even if the connection was unsuccessful, and with this PR the only way to ensure that `$TEMPDIR/$NODEIP.parse_tls_serverhello.txt` contains information from the most recent call to `parse_tls_serverhello()` is to ensure `$DEBUG -ge 1`.